### ISSUE TITLE
UI Modal Dialog

### ISSUE THREAD

**Issue by terrywbrady** [CDLUC3/mrt-doc#452]
Relates to #448 

Inadequate/inconsistent focus indicator for keyboard nav in Firefox and Chrome (MacOS); works fine in Safari. Focused element appears active rather than inactive, but lacks the browser-standard focus box. - Add standard focus indicator.

**Comment by terrywbrady** [comment]
@elopatin-uc3 , can you reproduce this issue?

**Comment by elopatin-uc3** [comment]
@terrywbrady If we're talking about the focus indicator around any element that's been tabbed to, I can't reproduce this. On the flip side, hitting tab in Chrome right after loading the Merritt home page shows the "Skip to main content" link at top center. But in FF and Safari, that doesn't happen. Safari just highlights it's own URL entry field, while repeated tabbing in FF moves from back button, to URL field, to bookmarks, and then a minimal highlight around the Merritt header (which you can't act upon with space or enter). I'm not worried about Safari per assumed low usage. Firefox I'm surprised at, and would be interested to hear your take about what's happening.

**Comment by terrywbrady** [comment]
@elopatin-uc3 , when I load merritt-stage in Firefox or Safari, I see skip to main content.

**Comment by terrywbrady** [comment]
This issue relates to our modal windows.  I see some unusual behavior in Firefox when tabbing to the dialog.

The behavior seems normal to me in Safari.  I would want to consult with the UX team for more details about the Safari issue
<img width="1263" alt="modal-focus" src="https://user-images.githubusercontent.com/1111057/92981707-e0d6f900-f44f-11ea-9410-ab017528c489.png">

I am going to leave this issue open for now.
